setup.py

```diff
@@ -356,6 +363,9 @@
     'obspy.plugin.inventory.CSS': [
         'writeFormat = obspy.io.css.station:_write_css',
         ],
+    'obspy.plugin.inventory.SEISAN': [
+        'writeFormat = obspy.io.nordic.core:inventory_to_seisan',
```

Comment:
If it is..
 - neither a well defined data format
 - nor used by a community or some widely used third-party program for I/O,
 - nor used by the rest of the PR (nordic file format I/O)..

..then I would say we should drop it.